src/vm/arm/ehhelpers.S

```diff
@@ -124,7 +128,9 @@ OFFSET_OF_FRAME=(4 + SIZEOF__GSCookie)
         // frame pointer for accessing the locals in the parent method.
         NESTED_ENTRY CallEHFilterFunclet, _TEXT, NoHandler
 
-        PROLOG_PUSH  "{lr}"
+        PROLOG_PUSH  "{r4-r11, lr}"
+        .setfp r7, sp, #12
```

Comment:
I would prefer you using the PROLOG_STACK_SAVE_OFFSET  macro that @myungjoo created in #4641. We prefer not using the unwinder instructions explicitly in the assembler helpers.